code/game/gamemodes/gang/recaller.dm

```diff
@@ -0,0 +1,454 @@
+					pointcost = 10
+			if("pistol")
+				if(gang.points >= 25)
+					item_type = /obj/item/weapon/gun/projectile/automatic/pistol
+					pointcost = 25
+			if("10mmammo")
+				if(gang.points >= 10)
+					item_type = /obj/item/ammo_box/magazine/m10mm
+					pointcost = 10
+			if("uzi")
+				if(gang.points >= 60)
+					item_type = /obj/item/weapon/gun/projectile/automatic/mini_uzi
+					pointcost = 60
+			if("9mmammo")
+				if(gang.points >= 40)
+					item_type = /obj/item/ammo_box/magazine/uzim9mm
+					pointcost = 40
+			if("scroll")
+				if(gang.points >= 30)
+					item_type = /obj/item/weapon/sleeping_carp_scroll
+					usr << "<span class='notice'>Anyone who reads the <b>sleeping carp scroll</b> will learn secrets of the sleeping carp martial arts style.</span>"
+					pointcost = 30
+			if("wrestlingbelt")
+				if(gang.points >= 20)
+					item_type = /obj/item/weapon/storage/belt/champion/wrestling
+					usr << "<span class='notice'>Anyone wearing the <b>wresting belt</b> will know how to be effective with wrestling.</span>"
+					pointcost = 20
+			if("bostaff")
+				if(gang.points >= 10)
+					item_type = /obj/item/weapon/twohanded/bostaff
+					pointcost = 10
+			if("C4")
+				if(gang.points >= 10)
+					item_type = /obj/item/weapon/c4
+					pointcost = 10
+			if("pen")
+				if((gang.points >= 50) || free_pen)
+					item_type = /obj/item/weapon/pen/gang
+					usr << "<span class='notice'>More <b>recruitment pens</b> will allow you to recruit gangsters faster. Only gang leaders can recruit with pens.</span>"
+					if(free_pen)
+						free_pen = 0
+					else
+						pointcost = 50
+			if("implant")
+				if(gang.points >= 15)
+					item_type = /obj/item/weapon/implanter/gang
+					usr << "<span class='notice'>The <b>implant breaker</b> is a single-use device that destroys all implants within the target before trying to recruit them to your gang. Also works on enemy gangsters.</span>"
+					pointcost = 15
+			if("gangtool")
+				if(gang.points >= 10)
+					if(usr.mind == gang.bosses[1])
+						item_type = /obj/item/device/gangtool/spare/lt
+						if(gang.bosses.len < 3)
+							usr << "<span class='notice'><b>Gangtools</b> allow you to promote a gangster to be your Lieutenant, enabling them to recruit and purchase items like you. Simply have them register the gangtool. You may promote up to [3-gang.bosses.len] more Lieutenants</span>"
+					else
+						item_type = /obj/item/device/gangtool/spare/
+					pointcost = 10
+			if("dominator")
+				if(!gang.dom_attempts)
+					return
+
+				var/area/usrarea = get_area(usr.loc)
+				var/usrturf = get_turf(usr.loc)
+				if(initial(usrarea.name) == "Space" || istype(usrturf,/turf/space) || usr.z != 1)
+					usr << "<span class='warning'>You can only use this on the station!</span>"
+					return
+
+				for(var/obj/obj in usrturf)
+					if(obj.density)
+						usr << "<span class='warning'>There's not enough room here!</span>"
+						return
+
+				if(usrarea.type in gang.territory|gang.territory_new)
```

Comment:
This might have a decent chance of breaking due to how shit the `in` operator is.